src/utils/update/fetch.beneficial.owners.private.data.ts

```diff
@@ -0,0 +1,52 @@
+import * as config from "../../config";
```

Comment:
maybe just import FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH instead of config.FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH
but minor improvement

Reply:
for some reason persistently made a test fail so kept it as is